e2e-tests/trade-finance-manager/cypress/e2e/journeys/case/amendments/amendment-tasks-manual.spec.js

```diff
@@ -432,7 +434,15 @@ context('Amendments tasks - manual amendment tasks', () => {
     pages.tasksPage.tasks.row(4, 4).status().contains('In progress');
     pages.tasksPage.tasks.row(4, 5).status().contains('Cannot start yet');
 
-    // TODO: DTFS2-6210 add when add decision is dependent on tasks.
+    cy.login(UNDERWRITER_MANAGER_1);
+    cy.visit(relative(`/case/${dealId}/underwriting`));
+    pages.underwritingPage.addAmendmentUnderwriterManagerDecisionButton().contains('Add decision');
+
+    cy.login(UNDERWRITER_MANAGER_1);
```

Comment:
In the backed up code, this is `PIM_USER_1`, but the 4th task is now for the team "Underwriter managers", so I've changed this to log in as an underwriter manager instead.